doc/governance/common_member_operations.rst

```diff
@@ -153,7 +153,7 @@ Renewing Node Certificate
 
 To renew the soon-to-be-expired certificate of a node, members should issue a ``set_node_certificate_validity`` proposal, specifying the date at which the validity period of the renewed certificate should start (``valid_from``), as well as its validity period in days (``validity_period_days`` -- optional).
 
-- The ``valid_from`` date argument should be a ASN1 UTCTime string, i.e. ``"YYMMDDhhmmssZ"``.
+- The ``valid_from`` date/time argument accepts time points in ASN.1 UTCTime format (``"YYMMDDhhmmssZ"``) or in the Python-default ISO 8601 format (``"YYYY-MM-DD HH:MM:SS.ssssss+HH:MM"``), with optional fractional seconds and timezone offset. For details see :ccf_repo:`src/ds/x509_time_fmt.h`.
```

Comment:
Is the ISO 8601 format that is now supported Python specific? If so, it's probably better to omit that detail here. Also, I think the link to `x509_time_fmt.h` isn't necessary is we list all supported formats here?

Reply:
No, ISO 8601 is an ISO standard and it's not one specific format, but a class of them, and Python chooses one of them as their default (e.g. `T` after date, before time is omitted, timezone offset is optional...). I'll remove `Python-default` here. 

We don't list all formats here as there are too many optional elements, so I figured I'd just add a link to the code.